packages/terra-date-picker/tests/jest/__snapshots__/DatePicker.test.jsx.snap

```diff
@@ -2,11 +2,10 @@ exports[`test should render a date picker with disabled dates 1`] = `
 <ResponsiveElement
   className="terra-DatePicker"
   defaultElement={
-    <DatePicker
+    <t
```

Comment:
I think it is the way the code in react-datepicker is being minified.

Check out `terra-core/packages/terra-datepicker/node_modules/react-datepicker/dist/react-datepicker.min.js`